server/publicapi/demo_data/data.json

```diff
@@ -0,0 +1,483 @@
+{
+    "_id": "tag:demodata.org,0001:ninjs_XYZ123",
+    "uri": "http://new.demodata.org/archive/20150316pic1234",
```

Comment:
BTW, what about the `href` attribute of the images' renditions objects? That should probably stay, right, because it might link to a resource outside the API database, e.g. to something in a CDN.